client/components/typography/typography.html

```diff
@@ -0,0 +1,20 @@
+<link rel="stylesheet" href="./typography.css">
+
+<h2 class="heading-h2">Page heading (h2)</h2>
+<h3 class="heading-h3">Enter details (h3)</h3>
+<button class="heading-h5">Button text</button>
+<input class="input heading-h5" value="Input value" />
+<input class="input heading-h5" placeholder="Input and Search placeholder" />
+<p class="input-label">Input label</p>
```

Comment:
if it's label, needs to be in correct tag <label>

Reply:
done